Validation/RecoEgamma/plugins/ElectronMcMiniAODSignalValidator.cc

```diff
@@ -15,13 +15,16 @@ typedef edm::Ptr<pat::Electron> PatElectronPtr;
 ElectronMcSignalValidatorMiniAOD::ElectronMcSignalValidatorMiniAOD(const edm::ParameterSet& iConfig) : ElectronDqmAnalyzerBase(iConfig)
 {
     mcTruthCollection_ = consumes<edm::View<reco::GenParticle> >(iConfig.getParameter<edm::InputTag>("mcTruthCollection")); // prunedGenParticles
-    electronToken_ = consumes<pat::ElectronCollection>(iConfig.getParameter<edm::InputTag>("electrons")); // slimmedElectrons
+    electronToken_     = consumes<pat::ElectronCollection>      (iConfig.getParameter<edm::InputTag>("electrons"));         // slimmedElectrons
```

Comment:
doesn't seem to be the case - look here (clicking the "view" button):

https://github.com/archiron/cmssw/blob/7926ced23e66b1dfeaef7a98b5e30c66cdd4d9fd/Validation/RecoEgamma/plugins/ElectronMcMiniAODSignalValidator.cc